scripts/pre_push_hook.py

```diff
@@ -200,6 +217,13 @@ def _start_sh_script(scriptname):
     return task.returncode
 
 
+def _has_uncommitted_files():
+    """Returns true if the repo contains modified files that are uncommitted.
+    Ignores untracked files."""
+    state = subprocess.check_output(GIT_IS_DIRTY_CMD.split(' '))
```

Comment:
I still think it's worth renaming "state" to something more descriptive of the output.